# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/moz.build
# Commit: e4783542b7c8
# Full Hash: e4783542b7c854ae4aa5cf60ac27e5e1ff19d56a
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2019-12-07 21:55:34
# Regressor Bug: 1588839
# File Overlap Count: 1
# Description:
#   Bug 1588839 - Part 1 - Add helper to allow structured cloning Error/Exception objects. r=bzbarsky
#   
#   In order to be able to reasonably debug error results from things like
#   JSWindowActor.sendQuery, we need to be able to clone errors across process
#   boundaries, so that they can be propagated to the caller that initiated a
# ==============================================================================

diff -r df371f905d3a -r e4783542b7c8 dom/ipc/moz.build
--- a/dom/ipc/moz.build	Sat Dec 07 17:45:53 2019 +0000
+++ b/dom/ipc/moz.build	Sat Dec 07 18:59:14 2019 +0000
@@ -40,6 +40,7 @@
     'BrowserChild.h',
     'BrowserHost.h',
     'BrowserParent.h',
+    'ClonedErrorHolder.h',
     'CoalescedInputData.h',
     'CoalescedMouseData.h',
     'CoalescedWheelData.h',
@@ -91,6 +92,7 @@
     'BrowserChild.cpp',
     'BrowserHost.cpp',
     'BrowserParent.cpp',
+    'ClonedErrorHolder.cpp',
     'CoalescedMouseData.cpp',
     'CoalescedWheelData.cpp',
     'ColorPickerParent.cpp',
